Neos.Neos/Tests/Unit/Fusion/PluginImplementationTest.php

```diff
@@ -96,12 +99,12 @@ public function responseHeadersDataProvider(): array
             [
                 'Plugin response key does already exist in parent with same value',
                 ['parent' => ['key' => 'value'], 'plugin' => ['key' => 'value']],
-                ['key' => 'value']
+                ['key' => 'value'] // 'value, value'
```

Comment:
> I think multiple competing headers like that are a broken use case anyways. Headers by definition can appear multiple times, we can't _really_ know if we should remove the first one and when not. IMHO the test is misleading the result might as well (correctly) be key => [value, value]

Reply:
Somehow the test now dont fail anymore ... but thats okay because as said the test are odd.